app/schema/api/current/models.rb

```diff
@@ -106,6 +106,32 @@ def nested_patient
       )
     end
 
+    def encounter
+      { type: :object,
+        properties: {
+          id: { type: :string,
+                format: :uuid,
+                pattern: '[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}',
+                description: 'This uuid is generated by hashing (SHA-1) the facility id, patient id, and encountered_on fields' },
+          deleted_at: { '$ref' => '#/definitions/nullable_timestamp' },
+          created_at: { '$ref' => '#/definitions/timestamp' },
+          updated_at: { '$ref' => '#/definitions/timestamp' },
+          patient_id: { '$ref' => '#/definitions/uuid' },
+          encountered_on: { type: :string, format: :date },
+          notes: { type: :string },
+          observations: {
+            type: :object,
+            properties: {
+              blood_pressures: { '$ref' => '#/definitions/blood_pressures' },
+            },
+            required: %w[blood_pressures]
```

Comment:
We can have empty observations, we should not have `blood_pressures` as a required field.